GitHubRepositories.ps1

```diff
@@ -461,6 +461,106 @@ function Get-GitHubRepository
     return Invoke-GHRestMethodMultipleResult @params
 }
 
+function Rename-GitHubRepository
+{
+<#
+    .SYNOPSIS
+        Rename a GitHub repository
+
+    .DESCRIPTION
+        Renames a GitHub repository with the new name provided.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER Uri
+        Uri for the repository to rename. You can supply this directly, or more easily by
+        using Get-GitHubRepository to get the repository as you please, and then piping the result to this cmdlet
+
+    .PARAMETER NewName
+        The new name to set for the given GitHub repository
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Get-GitHubRepository -Owner octocat -RepositoryName hello-world | Rename-GitHubRepository -NewName hello-again-world
+        Get the given 'hello-world' repo from the user 'octocat' and rename it to be https://github.com/octocat/hello-again-world.
+
+
+    .EXAMPLE
+        Get-GitHubRepository -Uri https://github.com/octocat/hello-world | Rename-GitHubRepository -NewName hello-again-world
+        Get the repository at https://github.com/octocat/hello-world and then rename it https://github.com/octocat/hello-again-world.
+
+    .EXAMPLE
+        Rename-GitHubRepository -Uri https://github.com/octocat/hello-world -NewName hello-again-world
+        Rename the repository at https://github.com/octocat/hello-world to https://github.com/octocat/hello-again-world.
+
+    .EXAMPLE
```

Comment:
Can you please add one example that passes in `-Confirm:$false` that shows how to call it silently in order to help out people (like me) who may not realize that it's possible to avoid that prompting if they really, truly want to?